Mongoose/Tests/Mongoose_Test_EdgeSeparator.cpp

```diff
@@ -65,8 +65,8 @@ int runEdgeSeparatorTest(const std::string &inputFile, const double targetSplit)
         int count = 0;
         for (int i = 0; i < result->n; i++)
         {
-            bool equals_0 = (result->partition[i] == 0);
-            bool equals_1 = (result->partition[i] == 1);
+            [[maybe_unused]] bool equals_0 = (result->partition[i] == 0);
```

Comment:
The `[[maybe_unused]]`` attribute was only added to the C++ standard in C++17, but the Mongoose CMake is configured to request C++11
https://github.com/DrTimothyAldenDavis/SuiteSparse/blob/fd8a7d1db63557a8262793f0c106f6e3dcf86581/Mongoose/CMakeLists.txt#L192
So I think this could lead to problems with some compilers if they are stricter with standard conformance.

Reply:
Good point -- thanks for the catch!

Mongoose only assumes c++11 so we cannot use that here.

Reply:
You are right. This is sad, I was happy to get this fixed for C++ so easily. Let us wait for C++17 and C23!